src/main/java/io/cdap/plugin/WindowAggregation.java

```diff
@@ -71,7 +71,7 @@ public class WindowAggregation extends SparkCompute<StructuredRecord, Structured
   public static final String NAME = "WindowAggregation";
   // BigQuery specific aggregations
   private static final Map<WindowAggregationConfig.Function, String> functionBQSqlMap =
-    new HashMap<WindowAggregationConfig.Function, String>() {{
+    new LinkedHashMap<WindowAggregationConfig.Function, String>() {{
```

Comment:
Why did you change it to a linked hash map?

Reply:
changed to hashmap